src/dmd/typesem.d

```diff
@@ -13,6 +13,7 @@
 module dmd.typesem;
 
 import core.checkedint;
+import core.stdc.stdio;
```

Comment:
remove ?

Reply:
No, because I keep having to add it back in.

Reply:
How about adding a `base` file that public imports `core.stdc.stdio` and other debug goodies when compiled in debug mode?

Reply:
Why make things more complicated? just import `core.stdc.stdio`

Reply:
Like a `debug` statement. But I do like the possibility to only enable debug output in specific areas in the code.